recipes/libgcc/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: libgcc
+  version: 5.2.0
```

Comment:
This has been a helpful reference for me: https://fedoramagazine.org/gcc-5-in-fedora-whats-an-abi-and-what-happens-when-we-change-it/